authcrypt/main.cpp

```diff
@@ -30,6 +30,8 @@ int main() {
     }
 
     int exit_code = MBEDTLS_EXIT_SUCCESS;
+    // The platform context is passed just in case any crypto calls need it.
+    // Please refer to https://github.com/ARMmbed/mbedtls/issues/1200 for more information.
```

Comment:
Please move this to the `Authcrypt` ctor, and have same comment like in the `hashing` and `benchmark` examples.